docs/source/reference/modding/extended.rst

```diff
@@ -0,0 +1,212 @@
+the optional third node represents the node state during daytime in interior cells).
+
+The behavior of such a model:
+
+1. During the day in exteriors, it is in the "normal" mode (child 0).
+
+2. During the night in exteriors, it is in the "night" mode (child 1).
+
+3. During the day in interiors, it is in the "normal" mode or "interior day" mode (child 2) depending on weather.
+
+4. During the night in interiors, it is in the "normal" mode.
+
+The actual state toggling time depends on the sunrise/sunset time settings in `openmw.cfg`:
+
+::
+
+    fallback=Weather_Sunrise_Time,6
+    fallback=Weather_Sunset_Time,18
+    fallback=Weather_Sunrise_Duration,2
+    fallback=Weather_Sunset_Duration,2
+
+These settings lead to the "night" starting at 20:00 and ending at 6:00.
+
+The engine checks if the weather is bright enough to support the "interior day" mode using the Glare_View setting. If it is >= 0.5, the engine considers the weather bright.
+
+::
+
+    fallback=Weather_Clear_Glare_View,1
+    fallback=Weather_Foggy_Glare_View,0.25
+
+With these settings, the "interior day" mode would be used for Clear weather, but would not be used for Foggy weather.
+
+Keep in mind that the engine will not update the weather type after a teleportation to a different region if the player did not move to an exterior cell in the new region yet.
+
+This feature can be used to implement street illumination, glowing windows, etc.
+
+Advantages of the described approach over old mods with glowing windows:
+
+1. There is no need to spawn additional objects for day and night mode
+
+2. There is no need to attach a script to every switchable object
+
+An example of a mod which uses this feature is `Glow in the Dahrk`_.
+
+
+Per-group animation files support
+---------------------------------
+
+In the original engine it is possible to add a custom animation file to NPC to override some animations (usually idle ones).
+In OpenMW it is possible to override animations via the same file for all actors which use a given basic animation file.
+
+If you want to override animations for all biped actors (which use the xbase_anim.nif skeleton), you can put your animations in the
+``Animations/xbase_anim`` folder in your ``Data Files``. You can also have them in a data folder with a higher priority.
+In this case any biped actor without a custom animation will use your animations, but – if he has additional animations – they have a higher priority.
+
+For example, all biped actors in Morrowind normally use the same spellcasting animations, so overriding xbase_anim spellcasting animations is sufficient.
+If you want to override walking animations, you should override ``xbase_anim_female`` and ``xbase_anim_kna`` animations -- these are used for women and beast races, and
+– because they have their own walking animations – they override ones which come from ``xbase_anim`` and its loose overrides.
```

Comment:
```suggestion
since they have their own walking animations, those will override ones from ``xbase_anim`` and its loose overrides.
```
Gah, this is confusing. :)